modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/test/java/com/liferay/dynamic/data/mapping/form/field/type/internal/numeric/NumericDDMFormFieldTemplateContextContributorTest.java

```diff
@@ -254,38 +245,51 @@ public void testGetPredefinedValue() {
 			"predefinedValue",
 			DDMFormValuesTestUtil.createLocalizedValue("42", _locale));
 
-		DDMFormFieldRenderingContext ddmFormFieldRenderingContext =
-			new DDMFormFieldRenderingContext();
-
-		ddmFormFieldRenderingContext.setLocale(_locale);
-		ddmFormFieldRenderingContext.setViewMode(true);
-
 		Map<String, Object> parameters =
 			_numericDDMFormFieldTemplateContextContributor.getParameters(
-				ddmFormField, ddmFormFieldRenderingContext);
+				ddmFormField, _createDDMFormFieldRenderingContext());
 
 		Assert.assertEquals(
 			"42", String.valueOf(parameters.get("predefinedValue")));
 	}
 
 	@Test
 	public void testGetSymbols() {
-		Map<String, String> symbolsMap =
-			_numericDDMFormFieldTemplateContextContributor.getSymbolsMap(
-				LocaleUtil.US);
+		DDMFormField ddmFormField = new DDMFormField("field", "numeric");
+
+		ddmFormField.setProperty("dataType", "double");
+
+		Map<String, Object> parameters =
+			_numericDDMFormFieldTemplateContextContributor.getParameters(
+				ddmFormField, _createDDMFormFieldRenderingContext());
 
-		Assert.assertEquals(".", symbolsMap.get("decimalSymbol"));
-		Assert.assertEquals(",", symbolsMap.get("thousandsSeparator"));
+		Map<String, String> symbols = (Map<String, String>)parameters.get(
+			"symbols");
+
+		Assert.assertEquals(".", symbols.get("decimalSymbol"));
+		Assert.assertEquals(",", symbols.get("thousandsSeparator"));
 	}
 
 	@Test
 	public void testGetSymbolsBrazilLocale() {
-		Map<String, String> symbolsMap =
-			_numericDDMFormFieldTemplateContextContributor.getSymbolsMap(
-				LocaleUtil.BRAZIL);
+		DDMFormField ddmFormField = new DDMFormField("field", "numeric");
 
-		Assert.assertEquals(",", symbolsMap.get("decimalSymbol"));
-		Assert.assertEquals(".", symbolsMap.get("thousandsSeparator"));
+		ddmFormField.setProperty("dataType", "double");
+
+		DDMFormFieldRenderingContext ddmFormFieldRenderingContext =
+			new DDMFormFieldRenderingContext();
+
+		ddmFormFieldRenderingContext.setLocale(LocaleUtil.BRAZIL);
+
+		Map<String, Object> parameters =
+			_numericDDMFormFieldTemplateContextContributor.getParameters(
+				ddmFormField, ddmFormFieldRenderingContext);
+
+		Map<String, String> symbols = (Map<String, String>)parameters.get(
```

Comment:
Shouldn't maps variables have the suffix `Maps`? [Reference](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#[inlineExtension]Maps-are-plurals,-with-suffix).